hand_eye_calibration/python/hand_eye_calibration/time_alignment.py

```diff
@@ -215,6 +215,8 @@ def interpolate_poses_from_samples(time_stamped_poses, samples):
 
   return aligned_poses.copy()
 
+def _filter_smaller_than(array, max_val):
+    return array[np.where(np.abs(array) < max_val)]
```

Comment:
The name is missing the `abs` in its implementation IMO -- even though "smaller" might make readers think of it. 
What about `_filter_magnitude_less_than` ?